ui/app/src/views/ViewDashboard.tsx

```diff
@@ -25,12 +26,15 @@ function ViewDashboard() {
     new URLSearchParams(window.location.search).get('dashboard') || DEFAULT_DASHBOARD_ID
   );
 
+  // TODO: Provider "current project" as argument to hook once we have proper routing
```

Comment:
Definitely need to discuss the plan for our URL structure at working session soon